internal/v1/handler_blueprints_test.go

```diff
@@ -1640,14 +1640,23 @@ func TestFixupBlueprint(t *testing.T) {
 	}))
 
 	cases := []struct {
-		blueprint  v1.BlueprintBody
+		createBody map[string]any
```

Comment:
With `map[string]any` I guess the above customizations aren't needed. But I'd prefer `v1.BlueprintBody` because that way we get type checking. To generate a valid upload request you can just use 
```
	var uo v1.UploadRequest_Options
	require.NoError(t, uo.FromAWSS3UploadRequestOptions(v1.AWSS3UploadRequestOptions{}))
```
and create it like
```
						UploadRequest: v1.UploadRequest{
							Type:    v1.UploadTypesAwsS3,
							Options: uo,
						},
```

also not sure if the rename from blueprint -> createBody makes it clearer